docs/en/latest/getting-started.md

```diff
@@ -23,36 +23,40 @@ title: Getting Started
 
 ## Summary
 
-This documentation is a quick start guide for Apache APISIX. The Quick Start is divided into the following three steps:
+This guide walks through how you can get up and running with Apache APISIX.
```

Comment:
This is a quick start guide to Apache APISIX. It will walk you through the basic steps to get up and running with Apache APISIX.

Reply:
Isn't the current change already capturing that intent of the guide?